packages/sw-lib/src/lib/error-factory.js

```diff
@@ -21,6 +21,9 @@ const errors = {
     'string, a Regex to capture request URLs or a Route instance.',
   'empty-express-string': 'The Express style route string must have some ' +
     'characters, an empty string is invalid.',
+  'bad-revisioned-cache-list': `The 'cacheRevisionedAssets()' method expects` +
```

Comment:
Agreed. I think it make sense before any notion of 1.0 release to have sweep of user niceties (i.e. ensure docs are available and flow well + good error messages). Main reason I opted for ErrorFactory is to somewhat make this easier.